ui/src/app/pages/workspace/workspace-library.spec.tsx

```diff
@@ -30,16 +30,10 @@ describe('WorkspaceLibrary', () => {
   let publishedWorkspaceStubs = [];
```

Comment:
Is this still needed or is the variable name unrelated to the abandoned feature that you mentioned?

Reply:
it is still needed, and it is related.

**Featured Workspaces** are a subset of **Published Workspaces**, and both will continue to exist.  Featured Workspaces are those Published Workspaces which are called out in `config/featured_workspaces_*.json` and are displayed on the Featured Workspaces page.

We had an idea once that we should also display the other Published Workspaces on that page too, under a category Published Workspaces.  That's kind of strange, and we never launched it.  That's the thing I'm deleting.